app/konnect/runtime-manager/runtime-groups/declarative-config.md

```diff
@@ -32,6 +32,12 @@ registration, or configure custom plugins.
 * decK v1.12.0 or later [installed](/deck/latest/installation/).
 * Optional: To test your configuration, [set up a simple runtime](/konnect/getting-started/configure-runtime).
 
+## Generate a Personal Access Token
+
+To test your connection, it is recommended that you use a personal access token (PAT).
```

Comment:
```suggestion
To test your connection, we recommend that you use a personal access token (PAT).
```
Going by what google does, and also just a personal preference. This is definitely a point of discussion for us though - what should the phrasing be, so that it's consistent across our docs? I stuck it into our S&G discussion list for now.